pywr-core/src/utils.rs

```diff
@@ -0,0 +1,33 @@
+/// Calculate the flow required to produce power using the hydropower equation
+pub fn inverse_hydropower_calculation(
+    power: f64,
+    water_elevation: f64,
+    turbine_elevation: f64,
+    efficiency: f64,
+    flow_unit_conversion: f64,
+    energy_unit_conversion: f64,
+    density: f64,
+) -> f64 {
+    let mut head = water_elevation - turbine_elevation;
```

Comment:
I would simplify this function to just accept `head: f64`. It's only use already calculates the head value and sets `turbine_elevation` to zero.